sdk/core/core-http/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/core-http",
   "sdk-type": "client",
   "author": "Microsoft Corporation",
-  "version": "1.2.4",
+  "version": "1.2.4-donotuse-not-actually-released",
```

Comment:
Let's comment why this is necessary as part of the PR description, it will help us backtrack in the future.

Reply:
I updated the PR description, however, I took a different route and just altered rush.json to remove any non-identity related packages (which wouldn't build without serious resyncing against 'master', which wouldn't be needed since we're not publishing any of those packages from this branch).